### DISCUSSION TITLE
Cascade onDelete with Explicit Many to Many relationship, doesn't seem to be working

### DISCUSSION THREAD

**Discussion by Deja967** [prisma/prisma#18137] [Q&A]
Not sure if this is a bug or what, I've read the documentation.. I cant find anything that suggest this shouldn't work. 

when I try to delete the record in the many to many join table, it deletes out of the join table but wont delete from the other "likes" table in this case

schema and code below

```
model Likes {
  id                 String   @id @unique
  createdAt    DateTime @default(now())
  updatedAt  DateTime @updatedAt
  userId         String   
  posts          LikesOnPosts[] 
  comments  LikesOnComments[]
}
model LikesOnComments {
  comment      Comment  @relation(fields: [commentId], references: [id], onDelete: Cascade)
  commentId  String 
  like               Likes?   @relation(fields: [likeId], references: [id], onDelete: Cascade) 
  likeId            String
  assignedAt   DateTime @default(now())
  assignedBy   String

  @@id([commentId, likeId])
}

```

in MySQL the likesOnComments record is not longer there it was deleted, but in the likes table that like is still there. I thought deleting likesOnComments would also delete the likes. I Could be missing something small

**Comment by Deja967** [comment]
nvm you can mark this as answered, I see it only works one way in the documentation

**Reply by nurul3101** [reply]
For reference, here's the [docs example](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#how-to-use-referential-actions) for referential action with explicit many to many.

**Comment by ChrisPlease** [comment]
Sorry, but where is this in the docs? And is there a workaround? AFAICT, this results in orphaned records in the db

**Comment by houselabss** [comment]
Any updates?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
